libsel4/include/sel4/bootinfo_types.h

```diff
@@ -75,11 +76,26 @@ typedef struct seL4_BootInfo {
      * to make this struct easier to represent in other languages */
 } seL4_BootInfo;
 
-/* If extraLen > 0, then 4K after the start of bootinfo there is a region of the
- * size extraLen that contains additional boot info data chunks. They are
- * arch/platform specific and may or may not exist in any given execution. Each
- * chunk has a header that contains an ID to describe the chunk. All IDs share a
- * global namespace to ensure uniqueness.
+/* The boot info frame must be large enough to hold the seL4_BootInfo data
+ * structure. Due to internal restrictions, the size must be of the form 2^n and
+ * the minimum is one page.
+ */
+#define BI_FRAME_SIZE_BITS  seL4_PageBits
+#define SEL4_BI_FRAME_SIZE  LIBSEL4_BIT(BI_FRAME_SIZE_BITS)
```

Comment:
This name should probably be seL4_BootInfoFrameSize to match the naming convention for other constants in libsel4. `BI_FRAME_SIZE_BITS` is a private kernel constant, and probably shouldn't be declared in libsel4 unless it was also named suitably.

Reply:
I will rename it to `seL4_BootInfoFrameSize`. And put another commit on top then that removes `BI_FRAME_SIZE_BITS` from the kernel code, as there seems no benefit keeping it as an alias.